components/axiosSidebar.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+module.exports = ({ url }) => `
+<div class="sidebar">
+  <h2>Mastering Axios</h2>
+  <ul>
```

Comment:
What do you think of  instead of linking to all the axios tutorials it instead randomly picks a few and displays those? That way it could maybe line up with the job board layout on the right? Or conversely make it smaller to end with the white space between the title and first paragraph?

Reply:
Also I just noticed but the buttons are a nice touch.

Reply:
I don't think random tutorials helps here, the goal is to have a path for learning axios and having a "previous" and "next" tutorial where possible. Random tutorials have no real guarantee of being relevant.

What does that have to do with lining up with the job board on the right?

Reply:
By line up I mean on the same x-coordinate. So since you're displaying all of them it goes past where the job board view ends.